### DISCUSSION TITLE
Can't build anything

### DISCUSSION THREAD

**Discussion by ijuedt** [wixtoolset/issues#7864] [Questions] [answered]
(1) When I run devbuild in the repo root, I first run into an "error MSB4096: The item "SfxCA.rc" in item list "ResourceCompile" does not define a value for metadata "Culture"." Googling in this case helps, as I am advised to add a Culture 0x0409 element to the ResourceCompile Include="SfxCA.rc" element.

(2) This being fixed, next error is "error MSB8020: The build tools for v143 (Platform Toolset = 'v143') cannot be found. To build using the v143 build tools, please install v143 build tools." Which doesn't apply to my system, as all is updated to the latest version of VS 2022, including v143 build tools. Stuck here.

(3) When opening any solution in VS 2022, I am greeted by a "This solution contains packages with vulnerabilities", accompanied by (in case of wix.sln) 101 "Unable to find package" errors. Trying to "Manage NuGet Packages" is of no avail: Manager flags 11 "Updates" and 1 "Consolidate", with a warning icon at "Installed". Any attempt to update provides the license dialogs to be expected, but update fails with the hint that the nuget package source is "not considered". Stuck here.

(4) In the discussion forum, I find the advice to build the tools solution first. Which reduces my nuget update errors to eight, but doesn't make a difference in principle. Stuck here.

How are users supposed to be successful here?

**Comment by barnson** [comment, marked as answer]
You're using VS 17.8, which was just released and broke the WiX build.

**Reply by ijuedt** [reply]
Ok, thanks for the reply, seems I was too quick with updating.

**Reply by chrpai** [reply]
Or not quick enough if you ask some people.  You have to be on 17.9 preview to see the latest .NET Aspire they are promoting.

**Comment by x86dev** [comment]
So what is the current solution to building this? I just installed latest VS 2022 (17.9.6) and still get the error for tags/4.0.5.
